types/node/index.d.ts

```diff
@@ -47,11 +47,13 @@
 //                 Yongsheng Zhang <https://github.com/ZYSzys>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-// NOTE: These definitions support NodeJS and TypeScript 3.7.
+// NOTE: These definitions support NodeJS and TypeScript 4.1.
 // Typically type modifications should be made in base.d.ts instead of here
 
 /// <reference path="base.d.ts" />
 
+/// <reference path="globals.dict.d.ts" />
+
 // NOTE: TypeScript version-specific augmentations can be found in the following paths:
 //          - ~/base.d.ts         - Shared definitions common to all TypeScript versions
 //          - ~/index.d.ts        - Definitions specific to TypeScript 2.8
```

Comment:
I wasn't sure whether to update the versions in the comments below. I don't really understand them because they are referencing a non-existent folder. There is no `ts3.5`.

Reply:
@sandersn switched most package definitions from referring to the oldest TS version in `index.d.ts` (with newer versions in `ts*` directories) to the latest TS version in `index.d.ts` (with older versions in `ts*` directories). It looks like this comment is just stale and needs to be updated.